scripts/train/celeba/cyclegan_celeba_preproc-256.py

```diff
@@ -209,6 +209,7 @@ def get_transfer_preset(cmdargs):
     'outdir' : os.path.join(ROOT_OUTDIR, 'celeba_preproc'),
     'log_level'  : 'DEBUG',
     'checkpoint' : 50,
+    'workers'    : 1,
```

Comment:
maybe put a comment saying `# for reproducibility purpose.` what do you think?

Reply:
Thank you @YHRen. Done in: [e9d7b31](https://github.com/LS4GAN/uvcgan/pull/1/commits/e9d7b31eafd05a367f69521564441c2cc9810013)